rakelib/plugin.rake

```diff
@@ -52,7 +52,18 @@ namespace "plugin" do
 
   task "install-all" do
     puts("[plugin:install-all] Installing all plugins from https://github.com/logstash-plugins")
-    install_plugins("--no-verify", *LogStash::RakeLib.fetch_all_plugins)
+    p = *LogStash::RakeLib.fetch_all_plugins
+    # Install plugin one by one, ignoring plugins that have issues. Otherwise, one bad plugin will
+    # blow up the entire install process.
+    # TODO Push this downstream to #install_plugins
+    p.each do |plugin|
+      begin
+        install_plugins("--no-verify", plugin)
+      rescue
```

Comment:
Not sure if the Gemfile will be in a consistent state if the one of the plugin fails, but its something we can/should monitor.

I think we will want to use Jenkins to run this command? We will need to make sure we bubble theses error up if we have too many fails this probably mean something is wrong with the build or the dependency tree?